react-with-graphql-tutorial.md

```diff
@@ -1880,46 +1881,44 @@ const resolveAddStarMutation = mutationResult => state => {
+* **Complementary:** To call a GraphQL API from your client application, you use HTTP methods. There are several quality libraries out there for HTTP requests, one of which is axios. That's why you have used axios for the previous application. However, using axios (or any other HTTP client library) doesn't feel like the best fit to complement a GraphQL centred interface. For instance, GraphQL doesn't use the full potential of HTTP. It's just fine to default to HTTP POST and only one API endpoint. It doesn't use resources and methods on those resources like a RESTful interface, so it makes no sense to specify a HTTP method and an API endpoint with every request, but to set it up once in the beginning instead. GraphQL comes with its own constraints. You could see it as a layer on top of HTTP when it's not as important for a developer to know about the underlying HTTP. A good developer should know about it anyway, though.
 
-* **Declarative:** Every time you make a query or mutation when using plain HTTP requests, you have to make a dedicated call to the API endpoint using a library such as axios. It's an imperative way of reading and writing data to your backend. However, what if there would be a declarative approach making queries and mutations? What if there would be a way to co-locate queries and mutations to your view-layer components? In the previous application, you have experienced how the query shape aligns perfectly with your component hierarchy shape. What if the queries and mutations would align in the same way? That's the power of co-locating your data-layer with your view-layer and you will find more about it when using a dedicated GraphQL client library for it.
+* **Declarative:** Every time you make a query or mutation when using plain HTTP requests, you have to make a dedicated call to the API endpoint using a library such as axios. It's an imperative way of reading and writing data to your backend. However, what if there was a declarative approach to making queries and mutations? What if there was a way to co-locate queries and mutations to your view-layer components? In the previous application, you experienced how the query shape aligned perfectly with your component hierarchy shape. What if the queries and mutations would align in the same way? That's the power of co-locating your data-layer with your view-layer, and uou will find more about it when you use a dedicated GraphQL client library for it.
```

Comment:
"and uou will find more about it when you use a dedicated GraphQL client library for it." to "and you will find more about it when you use a dedicated GraphQL client library for it."